server/src/main.ts

```diff
@@ -9,6 +9,7 @@ program.parse();
 const options = program.opts();
 
 const PORT_NUMBER = options.port || 4416;
+const VERSION = "0.4.0";
```

Comment:
```suggestion
const VERSION = "0.5.0";
```

Reply:
can we parse it from `package.json`?

Reply:
i think so

Reply:
related: https://stackoverflow.com/questions/41123631/how-to-get-the-version-from-the-package-json-in-typescript